ArtOfIllusion/src/artofillusion/MoveViewTool.java

```diff
@@ -181,7 +188,9 @@ private void dragMoveModel(WidgetMouseEvent e, ViewerCanvas view)
 		dx = dragPoint.x-clickPoint.x;
 		dy = dragPoint.y-clickPoint.y;
 
-		if (controlDown) // zoom!
+		
+
+		if (mouseButtonThree(e) && !mouseButtonTwo(e) && e.isControlDown()) // zoom!
```

Comment:
```suggestion
		if (controlDown) // zoom!
```

As above. If the `!mouseButtonTwo(e)`is actually needed, then something else must be broken: MB2 should have taken you to RotateViewTool.  If the user chooses to press two mouse buttons, the selection still happens in view level.

Reply:
Ditto...